server/app/schemas.py

```diff
@@ -243,7 +244,8 @@ class VectorToAdd(VectorBase):
     """
 
     date: str
-    genbank_content: str
+    genbank_content: Optional[str]
```

Comment:
Why distinguish this from `LevelNToAdd`? Also why change the naming convention?

Reply:
When we import level 1's we have a genbank file containing all the information. But when we construct one, using our tool, there is no genbank file and thus also no genbank content. So that's why I had to change it to optional.